frontend/spec/features/products_spec.rb

```diff
@@ -96,24 +98,22 @@
 
       it 'when adding a product to the cart', js: true do
         visit spree.product_path(product)
-        click_button 'Add To Cart'
-
-        within('#main-nav-bar') do
-          click_link 'Home'
+        add_to_cart(product) do
+          visit '/cart'
```

Comment:
It opens the cart by default
https://github.com/spark-solutions/spree/blob/db0f7a57c2d2fd4115dbc2d944ecae29c2d3d5c6/frontend/spec/support/add_to_cart.rb#L12